packages/clay-date-picker/src/Helpers.ts

```diff
@@ -3,7 +3,10 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import moment from 'moment';
+import formatDate from 'date-fns/format';
+import parseDate from 'date-fns/parse';
+
+export {formatDate, parseDate};
```

Comment:
If you write it like this, it becomes obvious that these are just re-exports and not used in this file:

```js
export {default as formatDate} from 'date-fns/format';
export {default as parseDate} from 'date-fns/parse';
```